src/components/dayCard/DayCard.js

```diff
@@ -0,0 +1,19 @@
+function DayCard({ date, day, imgAltText, imgSrc, temperature, description }) {
```

Comment:
nit: Very minor, but should this be alphabetised?

Reply:
It would be nice :-)